src/renderer/features/extension-wallet/model/wallets.ts

```diff
@@ -0,0 +1,54 @@
+import { type Wallet as ConnectWallet, getWallets } from '@talismn/connect-wallets';
+import { createEffect, createStore, sample } from 'effector';
+
+import { walletModel } from '@/entities/wallet';
+import { polkadotExtensionService } from '../service';
+
+import { extensionWalletFeature } from './feature';
+
+const $extensionWallets = createStore<ConnectWallet[]>([]);
+
+const $polkadot = walletModel.$wallets.map((wallets) =>
+  wallets.filter(polkadotExtensionService.isPolkadotExtensionWallet),
+);
+const $talisman = walletModel.$wallets.map((wallets) =>
+  wallets.filter(polkadotExtensionService.isTalismanExtensionWallet),
+);
+const $subWallet = walletModel.$wallets.map((wallets) =>
+  wallets.filter(polkadotExtensionService.isSubWalletExtensionWallet),
+);
+
+const $polkadotJsExtensionWallet = $extensionWallets.map(
+  (l) => l.find((e) => e.extensionName === 'polkadot-js') ?? null,
```

Comment:
Not clearly understand why do you use `l` here?

Reply:
list? 🌚 i'll fix that